NuRadioMC/EvtGen/NuRadioProposal.py

```diff
@@ -42,35 +42,44 @@
 pp_km = 1.e5
 
 
+def convert_energy_units(pp_energy):
```

Comment:
naming is not clear in which direction that goes... don't think this function actually helps

Reply:
yeah I agree...